service/templates/base-auth.html

```diff
@@ -35,6 +35,17 @@
   min-height: 100vh;
 }
 </style>
+<script type="module">
+  const form = document.querySelector('form');
+  const fields = form.querySelectorAll('input:not([hidden], select, textarea');
+
+  form.addEventListener('submit', (event) => {
+    /* To show user what was entered is what will be processed */
+    [...fields].forEach(field => {
+      field.setAttribute('readonly', '');
+    });
+  });
+</script>
```

Comment:
This is what actually locks the fields. The other change is to style them once locked.